integration/common_test.go

```diff
@@ -65,12 +63,23 @@ func eksctl(args ...string) *gexec.Session {
 
 func eksctlSuccess(args ...string) *gexec.Session {
 	session := eksctl(args...)
-	Expect(session.ExitCode()).To(Equal(0))
+	Expect(session.ExitCode()).To(BeZero())
 	return session
 }
 
 func eksctlFail(args ...string) *gexec.Session {
 	session := eksctl(args...)
-	Expect(session.ExitCode()).To(Not(Equal(0)))
+	Expect(session.ExitCode()).ToNot(BeZero())
 	return session
 }
+
+//eksctlStart starts running an eksctl command but doesn't wait for it to finish the command
+//This is primarily so that we can run eksctl create and then subsequently call eksctl delete
+//on the same cluster, but might be useful for other test scenarios as well.
+func eksctlStart(args ...string) (err error) {
```

Comment:
can remove return value since we assert it to be `nil`

Reply:
done.